vars/createDslContainers.groovy

```diff
@@ -64,7 +64,8 @@ def call(Map<String, ?> config=[:], Closure body){
             body()
         }
       }
-    } catch (FlowInterruptedException) {
+    } catch (FlowInterruptedException e) {
+        println(e)
```

Comment:
Not necessary. iirc this just states the exception type.

Reply:
It was helpful when debugging failures, but it's no longer necessary. I meant to remove it but I must have missed this one.